deploy/kubernetes/base/resources.yaml

```diff
@@ -205,6 +205,13 @@ rules:
     verbs:
       - get
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - update
```

Comment:
Add the `SYS_NAMESPACE` environment variable to `csi-node`, or use `config.Namespace` directly

https://github.com/juicedata/juicefs-csi-driver/pull/1218/files#diff-8a0b3df27b3a971a91992f9318d162dc3e0a16e72ce69382802ecda5a31a15b7L183-L190